pkg/api/client/client_internal.go

```diff
@@ -42,6 +43,19 @@ func (c *client) GetPublicKey(fingerprint, tenant string) (*models.PublicKey, er
 	return pubKey, nil
 }
 
+func (c *client) EvaluateKey(fingerprint string, dev *models.Device) (bool, error) {
+	var evaluate *bool
+	resp, _, errs := c.http.Post(buildURL(c, fmt.Sprintf("/internal/sshkeys/public-keys/evaluate/%s", fingerprint))).Send(dev).EndStruct(&evaluate)
+	if len(errs) > 0 {
+		return false, errs[0]
+	}
```

Comment:
Newline after this very first block.

Reply:
I don't know if we're using the [`multierr`](https://github.com/uber-go/multierr) package. If not, we might look at it.